MusicTaste/MusicTaste/MatchingAlgorithm.m

```diff
@@ -82,6 +89,17 @@ - (void) compareSpotifyData:(PFObject *)potentialMatch withData:(NSManagedObject
             }
     }
     //Saves matches to Database
+=======
+                    self.spotifyMatch = (((newGenreCount/genreCount)*0.2) + ((newTrackCount/trackCount)*0.3) + ((newAlbumCount/albumCount)*0.2) + ((newArtistCount/artistCount)*0.3));
```

Comment:
Cool, I see that you formatted the code. So you'll definitely want to pick this version of the code (while being sure not to lose any important changes in the other version).